sources/api/apiserver/src/server/ephemeral_storage.rs

```diff
@@ -269,18 +276,27 @@ pub fn ephemeral_devices() -> Result<Vec<String>> {
 }
 
 /// allowed_bind_dirs returns a set of the directories that can be bound to ephemeral storage, which
-/// varies based on the variant
-pub fn allowed_bind_dirs(variant: &str) -> HashSet<&'static str> {
-    let mut allowed = HashSet::from(["/var/lib/containerd", "/var/lib/host-containerd"]);
+/// varies based on the variant, a set of the prefixes of directories that are allowed to be bound.
+/// and a set of substrings that are disallowed in the directory name.
+pub fn allowed_bind_dirs(
+    variant: &str,
+) -> (
+    HashSet<&'static str>,
+    &'static [&'static str],
+    &'static [&'static str],
```

Comment:
nit: can we get these to be consistent and use `HashSet` here?
```suggestion
    HashSet<&'static str>,
    HashSet<&'static str>,
```

Reply:
pushed a commit to change to this